.github/workflows/node.yml

```diff
@@ -107,6 +107,7 @@ jobs:
       AWS_ENDPOINT: http://localhost:4566
       # this one is for dynamodb
       DYNAMODB_ENDPOINT: http://localhost:4566
+      ALLOW_HTTP: true
```

Comment:
We stopped hardcoding allowing unsafe connections to S3 in https://github.com/lancedb/lance/pull/2147, so have to set this explicitly now.